.github/workflows/cron-translations.yml

```diff
@@ -4,21 +4,28 @@ on:
   schedule:
     - cron: "7 0 * * *" # every day at 00:07 UTC
   workflow_dispatch:
+  push:
```

Comment:
Should we need to check whether the translation updated for every push? I think the schedule can work well.

Reply:
I don't think it is a bad idea, so I can update my translations directly after my PR merged for example 😅 I'm fine either ways.

Reply:
We previously had it pull translations on every push, which filled up the commit log with many small commits, so the balance was to sync down new translations once a day. We do have the workflow dispatch step to allow us to run the workflow on demand if there are any we must have immediately.

If that works with you:
```suggestion
```

Reply:
Okay, applied :)